plugin/metrics/README.md

```diff
@@ -32,6 +33,11 @@ Extra labels used are:
 * `type` which holds the query type. It holds most common types (A, AAAA, MX, SOA, CNAME, PTR, TXT,
   NS, SRV, DS, DNSKEY, RRSIG, NSEC, NSEC3, HTTPS, IXFR, AXFR and ANY) and "other" which lumps together all
   other types.
+* `scode` which holds the https status code. Possible values are:
```

Comment:
`scode` is non-standard. This should just be `status` to match the normal Prometheus naming convention.

Reply:
Fixed